doc/scapy/installation.rst

```diff
@@ -323,7 +323,6 @@ ImageMagick (takes long to compile)
  # tar xvfz /tmp/ports.tar.gz 
  # cd /usr/ports/graphics/ImageMagick/
  # make install
-
```

Comment:
Sorry, is now fixed.
(Went wrong while resolving merge conflict while using the GitHub webinterface.)